li-utils/src/main/java/com/linkedin/metadata/Constants.java

```diff
@@ -19,6 +19,8 @@ public class Constants {
   public static final String SYSTEM_ACTOR =
       "urn:li:corpuser:__datahub_system"; // DataHub internal service principal.
   public static final String UNKNOWN_ACTOR = "urn:li:corpuser:UNKNOWN"; // Unknown principal.
+  public static final Urn SYSTEM_POLICY_ZERO = UrnUtils.getUrn("urn:li:dataHubPolicy:0");
```

Comment:
I think we'd want this to be configurable via application.yaml so it could be overridden and/or extended.

Reply:
Hmm, as in the name of the policy? I think if it's going to be configurable the file should also be so that they change in lock-step. That's a little bit of a bigger change, can look into it. Right now it's a static value in a file that can't really be modified without code changes in which case this value would also need to change.